src/components/shared/VirtualList.js

```diff
@@ -217,9 +226,9 @@ type VirtualListProps<Item> = {|
   +itemHeight: CssPixels,
   +className: string,
   +renderItem: RenderItem<Item>,
-  +items: Item[],
+  +items: $ReadOnlyArray<Item>,
   +focusable: boolean,
-  +specialItems: Item[],
+  +specialItems: $ReadOnlyArray<Item | void>,
```

Comment:
Using a `$ReadOnlyArray` makes it possible to pass a tuple, thus making the type stricter on the call site.